src/main/java/com/monitorjbl/xlsx/impl/StreamingRow.java

```diff
@@ -85,6 +85,13 @@ public Cell createCell(int column, int type) {
   }
 
   /**
+   * Not supported.  Added for compatability with POI 3.12.
+   */
+  public int getOutlineLevel() {
```

Comment:
The comment seems to indicate that this is supposed to override a `Row` interface method that doesn't exist in the current version. Are you using a newer version of POI or something?